create_nginx_file.sh

```
@@ -0,0 +1,29 @@
+echo "Print current directory: $PWD" # Print current directory
+
+# mkdir
+for dir in node{0..3} seednode0; do
+    sudo mkdir -p "/data/nginx1/$dir"
+    sudo rm -rf "/data/nginx1/$dir"/*
+    sudo cp -a nginx_backup/* "/data/nginx1/$dir"
+done
+
+nodes=(node0 node1 node2 node3 seednode0)
+for node in "${nodes[@]}"; do
+    target_name="${node/testnet.treasurenet.io}" # 
```

Shellcheck (0.10.0)

[warning] target_name appears unused. Verify use (or export if used externally).

(SC2034)